lib/utils.js

```diff
@@ -330,6 +331,22 @@ const utils = module.exports = {
         let relativePath = path.relative(root, link).replace(/\\/g, '/');
 
         return '/' + relativePath;
-    }
+    },
+
+
+    /**
+     * Load a set of files
+     * @param  {string|array} dir
+     * @param  {string}       pattern
+     * @return {array}
+     */
+    loadFiles(dir, pattern) {
+        let files = [];
+        dir = !Array.isArray(dir) ? [dir] : dir;
```

Comment:
nit: I think it's easier to read this way:
```suggestion
        dir = Array.isArray(dir) ? dir : [dir];
```